gcc/rust/typecheck/rust-hir-type-check-item.cc

```diff
@@ -26,6 +26,8 @@
 #include "rust-substitution-mapper.h"
 #include "rust-type-util.h"
 
+#include <rust-tyty-variance-analysis.h>
```

Comment:
I think stick to single quotes for these includes like the others

Reply:
Fixed